packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -254,6 +254,68 @@ describe('<Slider />', () => {
       expect(container.firstChild).to.have.class(classes.disabled);
       expect(getByRole('slider')).to.not.have.attribute('tabIndex');
     });
+
+    const SelfDisablingSlider = () => {
```

Comment:
We try to keep test helpers local to the test since they tend to grow over time which makes debugging a specific test harder. Specifically here you don't need to re-use the component. The disabled-blur test can live with a standard `<Slider />` where you call `setProps({ disabled: true })` (returned from `render`).